I_have_no_vignette.py

```diff
@@ -0,0 +1,34 @@
+import random
+from datetime import date, timedelta, datetime
+import calendar
+
+
+def check_monday(date):
```

Comment:
1. please add documentation to the function
2. change its name and the variable name to something that will be more informative